.evergreen/config.yml

```diff
@@ -2533,6 +2547,18 @@ buildvariants:
     auth-ssl: "*"
   display_name: "${platform} ${python-version-windows} ${auth-ssl}"
   tasks: *all-server-versions
+  rules:
+    # Python 3.10 does not support TLS on MongoDB <= 3.4.
+    - if:
+        platform: "*"
+        python-version-windows: "3.10"
+        auth-ssl: "auth-ssl"
```

Comment:
I think we still want to test auth, right? Doesn't this skip tls and auth?

Reply:
Done.